lib/sidebar_page.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: false
+
+require 'jekyll'
+require_relative 'sidebar_path'
+require_relative 'sidebar_page_title'
+
+module SwedbankPay
+  # Represents a page in the Sidebar
+  class SidebarPage
+    attr_reader :path, :title, :parent, :level
+    attr_accessor :headers, :children, :doc
+
+    def initialize(page)
+      raise ArgumentError, 'Page must be a Jekyll::Page' unless page.is_a? Jekyll::Page
```

Comment:
Codacy found an issue: [Line is too long. [87/80]](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=6198407)